blocks/social-share/social-share.css

```diff
@@ -1 +1,4 @@
-/* empty */
\ No newline at end of file
+.social-share {
+margin-top: 2rem;
+margin-bottom: 4rem;
+}
```

Comment:
Not sure why the linter is not complaining. Please indent the properties inside the class.